### DISCUSSION TITLE
Adding my own gf

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#346] [Q&A]
I'm trying to add my own GF to the game but now it's being stupid
![image](https://github.com/user-attachments/assets/7cbba4ef-a30f-412f-a118-d4183c4b17e6)
Here's my file
![gf](https://github.com/user-attachments/assets/dc3c5c7e-0e61-4b8e-9d9b-50186d2edf6a)
I tried finding flxanimate/frames/whatever but I found nothing

**Comment by r6915ee** [comment]
Is it just supposed to be a PNG + XML pair? I don't use Adobe Animate, so I'm unfamiliar with how it functions and how it exports things.
